timemachine/cpp/CMakeLists.txt

```diff
@@ -92,7 +92,6 @@ pybind11_add_module(${LIBRARY_NAME} SHARED NO_EXTRAS
   src/pinned_host_buffer.cu
   src/streamed_potential_runner.cu
   src/stream_manager.cu
-  src/kernels/k_nonbonded.cu
```

Comment:
Deleted to avoid the difference in a single kernel that was made more difficult by templating said function.

Reply:
Makes sense. I think this was only a separate file before because `k_coords_to_kv_gather` had been untemplated, so putting it in a header file would lead to linker errors.